From ceb17f79f25a26dc18543a850af30a9e24e8dbb9 Mon Sep 17 00:00:00 2001
From: Hugo Osvaldo Barrera <hugo@whynothugo.nl>
Date: Fri, 16 Jun 2023 10:51:02 +0200
Subject: [PATCH] Remove bogus empty package

This empty package name is not valid, and results in `python setup.py
build` and `python setup.py wheel` failing with:

    WARNING: '' not a valid package name; please use only .-separated package names in setup.py

Removing this entirely fixes the issue completely.
---
 setup.py | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/setup.py b/setup.py
index 34dba94..fd273a8 100644
--- a/setup.py
+++ b/setup.py
@@ -18,8 +18,5 @@ setup(
             'subdl = subdl:cli'
         ]
     },
-    packages=[
-        ''
-    ],
     license='GPLv3+'
 )
-- 
2.41.0

